src/Symfony/Component/HttpKernel/DataCollector/LoggerDataCollector.php

```diff
@@ -106,6 +124,7 @@ private function sanitizeLogs($logs)
             if (isset($context['type'], $context['file'], $context['line'], $context['level'])) {
                 $errorId = md5("{$context['type']}/{$context['line']}/{$context['file']}\x00{$log['message']}", true);
                 $silenced = !($context['type'] & $context['level']);
+                $context['error_level'] = isset($this->errorLevel[$context['type']]) ? $this->errorLevel[$context['type']] : '';
```

Comment:
I'd write this as follows to just name it `name` and have it as first key as it belong to the type:

```
if (isset($this->errorNames[$context['type']])) {
                    $context = array_merge(array('name' => $this->errorNames[$context['type']]), $context);
                }
```